src/app/modal-content/modal-content.component.html

```diff
@@ -1,12 +1,13 @@
 <div class="modal-header">
   <h3 class="modal-title">{{title}}</h3>
-  <button type="button" class="close" aria-label="Close" (click)="activeModal.dismiss('Cross click')">
+  <button type="button" class="close" aria-label="Submitt another Ticket" (click)="(activeModal.dismiss())">
```

Comment:
Don't change this component for admin contact related content. This changes every modal on the entire site.